cloud/scaling-a-service.md

```diff
@@ -1,57 +1,36 @@
 # Resizing compute and storage in Timescale Cloud
 Timescale Cloud allows you to resize compute (CPU/RAM) and storage independently
-at any time. This is extremely useful when users have a need to increase storage
-(for instance) but not compute. The Timescale Cloud console makes this very easy
-to do for any service.
-
-Before you modify the compute or storage settings for a Cloud Service, please
-note the following limitations and when a change to these settings results
-in momentary downtime.
-
-**Storage**: Storage changes are applied with no downtime, typically available
-within a few seconds. Other things to note about storage changes:
-*   At the current time, storage can only be _increased_ in size.
-*   Storage size changes can only be made once every six (6) hours.
-*   Storage can be modified in various increments between 10GB and 10TB.
-
-**Compute**: Modifications to the compute size of your service (increases or
-decreases) can be applied at any time, however, please note the following:
-*   **_There is momentary downtime_** while the compute settings are applied.
+at any time. This is useful when you need to do something like increasing your
+storage capacity, but not your compute size. You can resize compute and storage
+in the Timescale Cloud console for any service, including multi-node
+environments.
+
+Storage changes are applied with no downtime, and the new storage capacity is
+usually available for use within a few seconds.
+*   Storage can only be increased in size. You cannot decrease the amount of
+    storage capacity your service has available.
+*   Storage size changes can only be made once every six hours.
+*   Storage can be modified in various increments between 10&nbsp;GB
+    and 16&nbsp;TB.
```

Comment:
After first reading this I had a feeling that 10GB and 16TB applies to increments rather than to the size. Maybe needs to be rephrased in a similar way?

```
Storage size can comprise from 10GB up to 16TB and can be changed in various increments.
```